test/statement_types.py

```diff
@@ -75,7 +75,7 @@ def cpp(self):
 
 class MatrixVariable(CppStatement):
     """Represents a matrix variable"""
-    def __init__(self, overload, name, stan_arg, value = None, size = None):
+    def __init__(self, overload, name, stan_arg, size, value = None):
```

Comment:
```suggestion
    def __init__(self, overload, name, stan_arg, size, value = 0.4):
```

Reply:
I tried switching to something like this but it doesn't work because `value = None` is used in `code_generator.py` ([here](https://github.com/stan-dev/math/blob/0e906cf97fc5b07354d6efaa381f646e95ec57dd/test/code_generator.py#L52)) to mean 'use the default value'.

Reply:
Ooooh, ok.